holoviews/plotting/bokeh/__init__.py

```diff
@@ -33,6 +33,9 @@
 
 Store.renderers['bokeh'] = BokehRenderer.instance()
 
+if len(Store.renderers) == 1:
```

Comment:
In that case, I would mention that in a script-level docstring at the top of the file. Same for the crossfilter app. It only needs to be one line explaining what these examples are supposed to show.